docs/source/installation/troubleshooting.rst

```diff
@@ -12,3 +12,62 @@ uses. Which can be done by running:
 .. code-block:: bash
 
   fmtutil -sys --all
+
+
+Installation does not support converting PDF to SVG?
+----------------------------------------------------
+
+First, make sure your ``dvisvgm`` version is at least 2.4:
+
+.. code-block:: bash
+
+  dvisvgm --version
+
+
+If you do not know how to update ``dvisvgm``, please refer to your operating system's documentation.
```

Comment:
Is there an official website for dvisgm? If so, please link it here

Reply:
OTOH, links might go stale and maintaining them up to date is a PITA...

Reply:
That's why I was hesitating. But I think we can take the risk and put a link, `dvisvgm` has been around for a while and probably will be.

Reply:
It's https://dvisvgm.de/
The download page is https://dvisvgm.de/Downloads/
This page doesn't look like it's going anywhere.

and hopefully very soon, nobody will actually trigger this error anymore as the installed default versions of dvisvgm are updated.